ansible_files/equinix_heterogeneous_create_infra_playbook.yml

```diff
@@ -84,6 +84,17 @@
       ansible.builtin.import_role:
         name: common/setup_ipip_tunnel
 
+- name: Check tunnel connectivity
+  hosts: all
+  tasks:
+    - name: Ping peer
+      ansible.builtin.command:
```

Comment:
not critical, but net-ping no good? https://docs.ansible.com/ansible/latest/collections/ansible/netcommon/net_ping_module.html#ansible-collections-ansible-netcommon-net-ping-module

Reply:
`ansible.netcommon.net_ping` looks to be a bit more specialized, it requires to set `ansible_network_os` in which linux is not, so I followed the advice:
`For targets running Python, use the ansible.builtin.shell module along with ping command instead.` :D